doc/phys_pkgs/gmredi.rst

```diff
@@ -443,7 +443,7 @@ The LDD97 tapering scheme is activated in the model by setting
 **GM\_taper\_scheme = ’ldd97’** in *data.gmredi*.
 
 
-.. figure:: figure_missing
+.. figure::
```

Comment:
This figure was missing in the original manual. I put this here as a place holder to check whether anyone knew anything about it. If nothing has surfaced by now, perhaps we should just delete it. @jm-c @christophernhill - do you know anything about this?

Reply:
Seems like nobody knows anything about this missing figure. Any objections to deleting it?